.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -11,16 +11,14 @@ You can see the action released in [here](https://github.com/marketplace/actions
 
 Checklist for a new release:
 
-- [ ] Create a PR that bumps the version in `package.json`
-  - [ ] Tag the sha with the same version `git tag _some_version_`
```

Comment:
Tagging a sha locally requires few more steps than using `git push` as part of the normal process of pushing changes to a PR. Since the UI let us tag `master` as part of the release creation, we can move that part of the process there.